### PR TITLE
Initial devise setup

### PR DESCRIPTION
**Pull request by ic3surf3r**
Initial set up of devise set up
Added nav bar
Set sign-in/out notification at the bottom right of screen

Screenshot 2023-11-20 at 4 19 24 pm

### CODE REVIEW

File: Gemfile.lock
Comment (Johnsteisel) [inline Gemfile.lock:1]: Why did you add bcrypt?

Comment (ic3surf3r) [reply Gemfile.lock:1]: Devise did it again

**Review by Johnsteisel — APPROVED** [review]
Well done João! So impressed with the setup you did ⭐⭐⭐!
